zerver/lib/message.py

```diff
@@ -591,7 +591,7 @@ def render_markdown(message: Message,
 
     return rendered_content
 
-def do_render_markdown(message: Message,
+def do_render_markdown(message: Any,
```

Comment:
I think it would be better to add a `type: ignore` or `cast(Message, `) to the `import_realm` call point, rather than changing the annotation here.

Reply:
I fixed with `cast`.